<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MOLUSCE</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInputs">
      <attribute name="title">
       <string>Inputs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QListWidget" name="lstLayers">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QPushButton" name="btnSetInitialRaster">
               <property name="text">
                <string>Initial &gt;&gt;</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="leInitRasterName"/>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="btnSetFinalRaster">
               <property name="text">
                <string>Final &gt;&gt;</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="leInitYear"/>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="leFinalRasterName"/>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="leFinalYear"/>
             </item>
             <item row="4" column="1" rowspan="5" colspan="2">
              <widget class="QListWidget" name="lstFactors">
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="8" column="0">
              <widget class="QPushButton" name="btnRemoveAllFactors">
               <property name="text">
                <string>&lt;&lt; Remove all</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QPushButton" name="btnRemoveFactor">
               <property name="text">
                <string>&lt;&lt; Remove</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QPushButton" name="btnAddFactor">
               <property name="text">
                <string>Add &gt;&gt;</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Factors driving landuse changes</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAreaChanges">
      <attribute name="title">
       <string>Area Changes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Class statistics</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Transition matrix</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="btnUpdateStatistics">
         <property name="text">
          <string>Update tables</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="btnCreateChangeMap">
         <property name="text">
          <string>Create changes map</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="MolusceTableWidget" name="tblStatistics"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="MolusceTableWidget" name="tblTransMatrix"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabModel">
      <attribute name="title">
       <string>Samples and Model</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="3">
        <widget class="QgsCollapsibleGroupBox" name="grpSampling">
         <property name="title">
          <string>Define Samples</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="collapsed" stdset="0">
          <bool>true</bool>
         </property>
         <property name="saveCollapsedState" stdset="0">
          <bool>true</bool>
         </property>
         <property name="saveCheckedState" stdset="0">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbSamplingMode"/>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Number of samples</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spnSamplesCount">
            <property name="maximum">
             <number>100000000</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Method</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cmbSimulationMethod">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QStackedWidget" name="widgetStackMethods">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSimulation">
      <attribute name="title">
       <string>Risk Simulation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" colspan="3">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="2">
          <widget class="QPushButton" name="btnSelectRiskFunction">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leRiskValidationPath">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="btnSelectRiskValidation">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="chkRiskFunction">
           <property name="text">
            <string>Risk function</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leRiskFunctionPath">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="chkRiskValidation">
           <property name="text">
            <string>Risk class validation</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="chkMonteCarlo">
           <property name="text">
            <string>Monte Carlo simulations</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="leMonteCarloPath">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="btnSelectMonteCarlo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="chkReuseMatrix">
           <property name="text">
            <string>Reuse input matrix</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="leMatrixPath">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="btnSelectMatrix">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="lblIterations">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Number of simulation iterations</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QSpinBox" name="spnIterations">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>94</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnStartSimulation">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMessages">
      <attribute name="title">
       <string>Messages</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextEdit" name="txtMessages">
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MolusceTableWidget</class>
   <extends>QTableWidget</extends>
   <header>molusce.moluscetablewidget</header>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
